runtime/devnet/src/config/contracts.rs

```diff
@@ -46,7 +35,7 @@ impl pallet_contracts::Config for Runtime {
 	type ApiVersion = ();
 	// IMPORTANT: only runtime calls through the api are allowed.
 	type CallFilter = Nothing;
-	type CallStack = [pallet_contracts::Frame<Self>; 23];
+	type CallStack = [pallet_contracts::Frame<Self>; 5];
```

Comment:
this could also potentially break already deployed contracts on testnet. Where does this change come from?